W-10877309 Exchange create custom policy for Flex Gateway jh (#188)
mulesoft/docs-exchange: modules/ROOT/pages/asset-types.adoc

* API Groups: A set of APIs bundled into a single asset
+
Instead of requesting access to multiple APIs to satisfy a use case, a developer can access the group in one step.

> * API Spec Fragments: A RAML document that has a version and an identifier but is not in itself a complete RAML specification

Reviewer: ```suggestion
* API Spec Fragments: A RAML document that has a version and identifier. This document is not a complete RAML specification.
```
